src/components/Profile/index.tsx

```diff
@@ -409,18 +425,41 @@ export const Profile = () => {
-                                <div className="flex w-full items-center justify-between">
-                                    <label>cyberdrk.eth</label>
-                                    <label>69000</label>
-                                </div>
-                                <div className="flex w-full items-center justify-between">
-                                    <label>kkonrad.eth</label>
-                                    <label>420</label>
-                                </div>
+                                {user?.referredUsers > 0 &&
+                                    user?.totalReferralConnections.map((referral, index) => (
+                                        <div key={index} className="flex w-full items-center justify-between">
+                                            <label>{utils.shortenAddressLong(referral.account_identifier)}</label>
+                                            <label>
+                                                {Math.floor(
+                                                    user.pointsPerReferral?.find((ref) =>
+                                                        utils.compareTokenAddresses(
+                                                            ref.address,
+                                                            referral.account_identifier
+                                                        )
+                                                    )?.points ?? 0
+                                                )}
+                                            </label>
+                                        </div>
+                                    ))}
+
                                 <Divider borderColor={'black'}></Divider>
                                 <div className="flex w-full items-center justify-between">
                                     <label>Total</label>
-                                    <label>69420</label>
+                                    <label>
+                                        {user?.totalReferralConnections.reduce((acc, referral) => {
+                                            return (
+                                                acc +
+                                                Math.floor(
+                                                    user.pointsPerReferral?.find((ref) =>
+                                                        utils.compareTokenAddresses(
+                                                            ref.address,
+                                                            referral.account_identifier
+                                                        )
+                                                    )?.points ?? 0
+                                                )
+                                            )
+                                        }, 0)}
+                                    </label>
```

Comment:
Nit: this seems a bit complex/confusing and i can't tell from a first glance what it is for. Would be useful to have a 1 line comment explaining what the label is for

Reply:
IMO its not that complicated, just reducing the points in an array to get the total points. 

Also, dont like putting comments in html code, makes it v messy